src/java.base/share/classes/java/util/concurrent/ForkJoinPool.java

```diff
@@ -2088,84 +2091,83 @@ else if (o == null)       // contended
     }
 
     /**
-     * Deactivates w, reactivating on contention, already signalled,
-     * or possible missed signal.
+     * Tries to inactivate, and if successful, awaits signal or termination.
      *
      * @param w the worker (may be null if already terminated)
```

Comment:
Technically `w` is `WorkQueue` not `ForkJoinWorkerThread` so perhaps s/worker/work queue?